### ISSUE TITLE
stateMutability and Visibility can't invoke

### ISSUE THREAD

**Issue by seongeunC** [ConsenSys/python-solidity-parser#4]
I can not invoke the statemutability and visibility of individual functions in a contract. Could you tell me how?

**Comment by tintinweb** [comment]
Hi @seongeunC,

you can access it via the `_node` attribute.
```python
objectified_contract_object.functions[functionName]._node.visibility
objectified_contract_object.functions[functionName]._node.stateMutability
```

**Comment by seongeunC** [comment]
Thank you for answer. Great help!

**Comment by tintinweb** [comment]
@seongeunC 

released and published v0.0.5 that gives you direct access to `objectified_contract_object.functions[functionName].visibility` and `objectified_contract_object.functions[functionName].stateMutability` (or see this example: https://github.com/ConsenSys/python-solidity-parser/blob/master/solidity_parser/__main__.py#L66-L72)

Let me know if there's any other property you might find useful to be exposed directly in the function object.

cheers,
tin
